core/swift3.1.1Action/spm-build/Package.swift

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2015-2016 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PackageDescription
+
+let package = Package(
+    name: "Action",
+        dependencies: [
+            .Package(url: "https://github.com/IBM-Swift/Kitura-net.git", "1.7.6"),
```

Comment:
@paulcastro @rabbah confirmed we have to pinned to 0.2.3, using 0.2.4 we have test failures
same tests are failing as for swift3
```whisk.core.cli.test.Swift311Tests.Swift 3 Whisk SDK tests should allow Swift actions to invoke other actions
whisk.core.cli.test.Swift311Tests.Swift 3 Whisk SDK tests should allow Swift actions to invoke other actions and not block
whisk.core.cli.test.Swift311Tests.Swift 3 Whisk SDK tests should allow Swift actions to trigger events
whisk.core.cli.test.Swift3Tests.Swift 3 Whisk SDK tests should allow Swift actions to invoke other actions
whisk.core.cli.test.Swift3Tests.Swift 3 Whisk SDK tests should allow Swift actions to invoke other actions and not block
whisk.core.cli.test.Swift3Tests.Swift 3 Whisk SDK tests should allow Swift actions to trigger events
```